docs/src/main/asciidoc/functional.adoc

```diff
@@ -292,28 +292,21 @@ And to help with correct dependencies here is the excerpt from POM
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-test</artifactId>
         <scope>test</scope>
-        <exclusions>
```

Comment:
Minor thing, but no longer required as recent boot versions don't carry this baggage anymore.